.github/workflows/publish-to-npm.yml

```diff
@@ -38,4 +38,4 @@ jobs:
         # (packages are in-fact published in the correct order), but the race on the registry still applies.
         # If this happens, run the workflow again - there should be enough time for everything to settle until this workflow
         # attempts to publish again.
-        run: npx lerna publish --concurrency 1 from-package --no-push --no-private --no-git-tag-version --no-verify-access --yes
+        run: npx lerna publish --concurrency 1 from-package --no-push --no-private --no-git-tag-version --no-verify-access --dist-tag=canary --yes
```

Comment:
@pichlermarc I notice there is a second `lerna publish ...` call lurking in package.json scripts:

https://github.com/open-telemetry/opentelemetry-js/blob/492ed355cb4becedbfaa00d7737c5c40c9132754/package.json#L46

I think that vestigial. We should remove that, right?

Reply:
@trentm, I had kept this in case we ever have to publish manually. In that case though, we should just take whatever the workflow is doing and not this. 

I'll remove it 👍

Reply:
removed in [bdb0e18](https://github.com/open-telemetry/opentelemetry-js/pull/5496/commits/bdb0e18891bea33d82d0e6ecc043e70750dde126)